### ISSUE TITLE
Crash on Mysterious Beam, modified by "Infinite"

### ISSUE THREAD

**Issue by petco130** [scarf005/Marisa#212]
### How to reproduce

I tried to play "Infinite" Mysterious Beam, infinite coming from Chimera Cards, and it crashes consistently on this.

### Crash log

java.lang.NullPointerException: Parameter specified as non-null is null: method marisa.cards.MysteriousBeam.calculateCardDamage, parameter mo
	at marisa.cards.MysteriousBeam.calculateCardDamage(MysteriousBeam.kt)
	at com.megacrit.cardcrawl.characters.AbstractPlayer.useCard(AbstractPlayer.java:1685)
	at com.megacrit.cardcrawl.actions.GameActionManager.getNextAction(GameActionManager.java:359)
	at com.megacrit.cardcrawl.actions.GameActionManager.update(GameActionManager.java:175)
	at com.megacrit.cardcrawl.rooms.AbstractRoom.update(AbstractRoom.java:325)
	at com.megacrit.cardcrawl.dungeons.AbstractDungeon.update(AbstractDungeon.java:2532)
	at com.megacrit.cardcrawl.core.CardCrawlGame.update(CardCrawlGame.java:876)
	at com.megacrit.cardcrawl.core.CardCrawlGame.render(CardCrawlGame.java:423)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:225)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126)

**Comment by scarf005** [comment]
hello, thanks for the bug report!
I apologize in advance that the issue may take quite some time as I'm really busy and not familiar with bugs involving other mods. Some more information would be helpful:

1. does the error only happen on mysterious beam or for other cards?
2. does the behavior dependent on mysterious beam / infinite being upgraded?

**Comment by petco130** [comment]
Not a problem; I think my run is probably utterly demolished anyways, so no big loss.

1. It seems to only happen in this instance, but I haven't seen many instances of this problem at all, given the randomness of the scenario.
2. Unclear if it's about it being upgraded, though it is upgraded.

Infinite turns any card into a power, that adds a copy of the card (upgraded or otherwise) to your hand every turn.

**Comment by scarf005** [comment]
I see. Were you able to reliably reproduce the issue via [console command](https://alexdriedger.github.io/SlayTheSpireModding/docs/console)? It would be very helpful in diagnosing the error.

**Comment by petco130** [comment]
I did not know that was an option; I've already uninstalled the mod, unfortunately.

**Comment by scarf005** [comment]
I'm sorry to hear that. Did you uninstall this mod, or the chimera cards? does either work well without anohter?

**Comment by petco130** [comment]
I just installed the Marisa mod; nothing personal or anything, I'm more of a taste tester kind of gamer, at least for StS mods.

I think it had to be the interaction; I had played Marisa by itself before, and Chimera didn't seem to have this interaction elsewhere. I presume it was just the mixing of the two that was the problem.
    On Wednesday, June 28, 2023 at 08:17:53 PM EDT, scarf ***@***.***> wrote:  
 
 


I'm sorry to hear that. Did you uninstall this mod, or the chimera cards? does either work well without anohter?
